fhirclient/server.py

```diff
@@ -184,6 +187,7 @@ def _get(self, path, headers={}, nosign=False):
         
         # perform the request but intercept 401 responses, raising our own Exception
         res = self.session.get(url, headers=headers)
+        logging.debug('SMART SERVER:\r\n{0}'.format(dump.dump_response(res).decode('utf-8')))
```

Comment:
Is there a reason you're using the root logger here instead of the new `logger`? Also, please don't use `\r\n`, use either `\n` or `os.linesep`.